rpcserver.go

```diff
@@ -3965,8 +3965,7 @@ func handleGetStakeVersions(s *rpcServer, cmd interface{}, closeChan <-chan stru
 		}
 		for _, vote := range v.Votes {
 			nsv.Votes = append(nsv.Votes,
-				dcrjson.VersionBits{Version: vote.Version,
-					Bits: vote.Bits})
+				dcrjson.VersionBits(vote))
```

Comment:
This conversion is not supported by Go 1.7.x which we still support, so this can't go in.